src/components/interactive-builder/edit-question.modal.tsx

```diff
@@ -875,7 +876,14 @@ const EditQuestionModal: React.FC<EditQuestionModalProps> = ({
                       </div>
                     ) : null}
 
-                    {(selectedConcept || questionToEdit) && questionToEdit?.questionOptions.answers?.length ? (
+                    {!hasConceptChanged && editConceptInfo?.datatype?.uuid == '8d4a48b6-c2cc-11de-8d13-0010c6dffd0f' ? (
```

Comment:
Why do we have a hard-coded concept UUID?

Reply:
@Willie-theBeastMutua please check my comment, just removing the conditional logic is sufficient, and we'll merge this in. Thanks!

Reply:
> Why do we have a hard-coded UUID?

The UUID id for the coded datatype that is the check

Reply:
> @Willie-theBeastMutua please check my comment, just removing the conditional logic is sufficient, and we'll merge this in. Thanks!

If I remove the logic, the more answers will display even when the concept is not of a coded datatype so I added a hook that collects data on the concept and I check for the UUID for the coded datatype. I can use the word "Coded" to check but my fear was what would happen if the language changed

Reply:
Ignore my suggestion about simply removing the condition, I see why the hook is needed. 
@ibacher if we do a datatype.display === "Coded", would that be language sensitive?

Reply:
@Willie-theBeastMutua for now lets assume it wont change with the language

Reply:
@NethmiRodrigo Yes, but if you used `datatype.name` is shouldn't be, except in some edge-cases.

Reply:
In general `display` is the (possibly) localized metadata, but `name` is the "raw" value.